ratelimit/utils.py

```diff
@@ -187,8 +187,12 @@ def get_usage_count(request, group=None, fn=None, key=None, rate=None,
 
 
 def is_authenticated(user):
-    # is_authenticated was a method in Django < 1.10
-    if callable(user.is_authenticated):
+    if getattr(user.is_authenticated, 'do_not_call_in_templates', False):
```

Comment:
I think it's easier to do:

```python
if django.VERSION > (1, 10):
   return user.is_authenticated
else:
    return user.is_authenticated()
```

Then it's also obvious which branch needs dropping on older Django versions